web/cypress/integration/Home/CreateTest.spec.ts

```diff
@@ -24,7 +24,6 @@ describe('Create test', () => {
 
     cy.location('pathname').should('match', /\/test\/.*/i);
     cy.get('[data-cy=test-details-name]').should('have.text', `${name} (v1)`);
-    deleteTest();
```

Comment:
Are we deleting the tests in other place? We should delete them after each `test/spec`.

Reply:
I must have removed that when working on that. My bad, will fix it now